src/components/popup/Route.tsx

```diff
@@ -26,10 +26,9 @@ export const Wrapper = styled(motion.div)<{
   expanded?: boolean;
 }>`
   position: relative;
-  width: ${(props) => (props.responsive ? "100%" : "377.5px")};
```

Comment:
This `.5` seem to fix some rendering issues (underflow) in some cases but cause another issue (overflow) in some other cases. I'd just leave it as `377px` which is the actual value this should have everywhere and fix it later by laying out the components differently.